src/_versions.ts

```diff
@@ -9,10 +9,10 @@ export interface TsAppVersion {
     gitTag?: string;
 };
 export const versions: TsAppVersion = {
-    version: '0.1.1',
+    version: '0.1.0',
```

Comment:
why was the version downgraded?

Reply:
I am not too sure what happened here some file I don't recognised got changed and I don't have a description I might have opened this by mistake or something I will fix it

Reply:
open an issue to automate validation that versions is provided consistently in our typescript projects and that the version is incremented between PR.

Reply:
changed